### ISSUE TITLE
Change granularity unit to 100 instead of 1

### ISSUE THREAD

**Issue by tnmendes** [edgar-zigis/GaugeSlider#2]
First of all, I want to congratulate you for the great work with this slider.

I'm trying to adapt this slider to use to select the color temperature for a smart lamp. The temperature of the lamps goes from 2500 K to 9000 K. Is there any way when you are sliding instead of advancing unit to unit advance but rather every 100.

Example Current:
2500 > 2501 > 2502

What I want:
2500 > 2600 > 2700

**Comment by edgar-zigis** [comment]
Yeah, I get what you want. Probably Google Assistant (or which integration you use) can't cope with such multiple minor value changes. For this particular reason I did create `delegationMode` variable so you could get immediate result or just after you finish sliding. 

I will put a setting to immediate, so you could set it like `delegationMode = .immediate(granularity: 100)` (with default == 1).

But just in case, you can reach the same effect with RxSwift:

```swift
private let progressValue = Variable<Int>(0)

gaugeSlider.onProgressChanged = { [weak self] progress in
    self?.progressValue?.value = progress
}

progressValue.asObservable()
    .skip(1)
    .throttle(0.15, scheduler: MainScheduler.instance)  //  depending on throttle value you will get different interval
    .distinctUntilChanged()
    .subscribe({ (value) in
        //  do whatever you want with value
    }).disposed(by: disposeBag)
```

**Comment by tnmendes** [comment]
My goal with granularity to improve the user usability, special the final part of the slide after 8930 sometimes is a bit tricky to reach the 9000.

Yah add a new parameter to .immediate that sounds nice :)

**Comment by tnmendes** [comment]
![IMG_2750](https://user-images.githubusercontent.com/9822076/87436240-ab936380-c5e4-11ea-8358-32f949f5c6f0.jpg)

**Comment by edgar-zigis** [comment]
@tnmendes released under version 1.2.1
Just use `delegationMode = .immediate(interval: 1000)`

**Comment by tnmendes** [comment]
Thank you!!! 👍
